.github/ISSUE_TEMPLATE/20_feature-request.yml

```
@@ -0,0 +1,35 @@
+name: 💡 Feature request
+description: Suggest an idea for this project
+body:
+  - type: markdown
+    attributes:
+      value: |
+        More information on our issue management policies can be found here: https://aka.ms/aspnet/issue-policies
+  - type: checkboxes
+    attributes:
+      label: Is there an existing issue for this?
+      description: Please search to see if an issue already exists for the feature you are requesting. (https://github.com/dotnet/aspnetcore/issues).
```

```suggestion
      description: Please search to see if an issue already exists for the feature you are requesting. (https://github.com/dotnet/aspire/issues).
```
I really should not have reviewed on my phone ..

---

Found already and fixed ;-)